csaf_2.1/prose/edit/src/conformance.md

```diff
@@ -256,6 +256,8 @@ A CSAF content management system satisfies the "CSAF content management system"
     the configuration (default: 3 weeks)
   * suggest to publish a new version of the CSAF document with the document status `final` if the document status was
     `interim` and no new release has be done during the given threshold in the configuration (default: 6 weeks)
+    > Note that the terms "publish", "publication" and their derived forms are used in this conformance profile independent of
+      whether the intended target group is the public or a closed group.
```

Comment:
Again: Can we say specified?

Reply:
[39a992a](https://github.com/oasis-tcs/csaf/pull/873/commits/39a992af59e2a47933a360e9724a3ff682af5e9c)